Newsletters/_posts/Yearn-Finance-Newsletter-43/es.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Newsletter #43 de Yearn Finance"
+categories: [ Newsletters ]
+image: ./Yearn-Finance-Newsletter-43/image1.jpg
+author: Yearn
+translator: RestlessMik3
+publish: true
+---
+
+# Newsletter #43 de Yearn Finance
+
+![](image1.jpg)
+
+Bienvenido a la 43.ª edición de la newsletter de Yearn Finance. Nuestro objetivo con esta newsletter es mantener a la comunidad de Yearn y a la comunidad cripto informada sobre las últimas noticias, incluidos los lanzamientos de productos, los cambios en la gobernanza y las actualizaciones del ecosistema. Si estás interesado en obtener más información sobre Yearn Finance, sigue nuestras cuentas oficiales de [Twitter](https://twitter.com/iearnfinance) y [Medium](https://medium.com/iearn).
+
+## **Resumen**
+
+- YIP-63: Financiar DAO de activismo legal centrado en contructores
+- Alchemix migra a Yearn v2
+- YFI añadido a Compound Finance
+- Intercambiar entre vaults de Yearn en una transacción
+- Vaults en Yearn
+- Noticias del ecosistema
+
+## **YIP-63: Fund Builder-First Legal Activism DAO**
+
+![](image2.jpg)
+
+En medio de un escrutinio regulatorio cada vez más intenso de DeFi, se ha propuesto que LeXpunK_DAO debería dedicarse a la defensa legal para Yearn y otras comunidades DeFi centradas en constructores. El LeXpunK_DAO estará gobernado por constructores de comunidades contribuyentes (incluido Yearn) y abogados en ejercicio del LeXpunK Army 146.
```

Comment:
AS-IS: En medio de un escrutinio regulatorio cada vez más intenso de DeFi, se ha propuesto que LeXpunK_DAO debería dedicarse a la defensa legal para Yearn y otras comunidades DeFi centradas en constructores. El LeXpunK_DAO estará gobernado por constructores de comunidades contribuyentes (incluido Yearn) y abogados en ejercicio del LeXpunK Army 146.


TO-BE:En medio de un escrutinio regulatorio cada vez más intenso de DeFi, se ha propuesto que LeXpunK_DAO debería dedicarse a la defensa legal de Yearn y otras comunidades DeFi centradas en constructores. El LeXpunK_DAO estará gobernado por constructores de comunidades contribuyentes (incluido Yearn) y abogados en ejercicio del LeXpunK Army 146.

Reply:
Done